### ISSUE TITLE
Remove the custom migration logic from all runtimes

### ISSUE THREAD

**Issue by ebma** [pendulum-chain/pendulum#454]
As part of our efforts to upgrade the dependencies to v0.9.42, we added custom migration logic to all runtimes. Now that these changes are rolled out and all the migrations applied, we can remove that extra logic again.

## TODO
- Remove the `CustomOnRumtimeUpgrade` implementations again
- Remove the customized dependencies. We had to create forks of some projects in order to be able to modify their migration logic. We can delete those forks again and switch back to the 'official'/upstream dependency versions.

**Comment by ebma** [comment]
@pendulum-chain/product this is more of a chore but nice-to-have and low effort. We can clean up some unnecessary code and repositories with this.

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/pendulum-chain/pendulum/454?planning-poker) with Zenhub @b-yap @bogdanS98 @ebma @gianfra-t @TorstenStueber

**Comment by gianfra-t** [comment]
I think we can reuse this ticket and remove the custom migration logic after the [1.1.0 upgrade](https://github.com/pendulum-chain/pendulum/issues/482) is completed. Otherwise we would need to re-add them soon.

**Comment by ebma** [comment]
Right, let's do that. I added the ticket as a blocker to the Zenhub dependencies.

**Comment by vadaynujra** [comment]
@ebma is this possible to be worked on now, or only after the upgrade on Pendulum?

**Comment by gianfra-t** [comment]
@vadaynujra only once we have updated the runtimes we can remove this extra code. We left this ticket also as a reminder to remove the extra logic after the runtime upgrade that bumps to  `1.1.0` is enacted on all runtimes. 

In any case, this is not blocking anything.
